gradle/maven.gradle

```diff
@@ -60,12 +60,21 @@ plugins.withId('maven') {
                     }
                 }
 
-                // Sign the POM when the signing plugin is present
+                // Sign the POM when the signing plugin is present.
+                // If Gradle version < 7.0
                 plugins.withId('signing') {
                     if (project.hasProperty('doSigning')) {
                         beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
                     }
                 }
+
+                // Sign the POM when the signing plugin is present.
+                // If Gradle version > 7.0
```

Comment:
@davideduma, can you clean up the PR by removing the commented codes?

Reply:
Yes. Implemented.